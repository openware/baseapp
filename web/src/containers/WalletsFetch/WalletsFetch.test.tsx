import { render } from '@testing-library/react';
import React from 'react';
import { TestComponentWrapper } from 'src/lib/test';
import { IntlProps } from '../../';
import { WalletsFetch } from './';

const renderComponent = (props: Partial<IntlProps> = {}) =>
    render(
        <TestComponentWrapper>
            <WalletsFetch />
        </TestComponentWrapper>,
    );

describe('WalletsFetch component', () => {
    it('should match snapshot', () => {
        expect(renderComponent().container).toMatchSnapshot();
    });
});
