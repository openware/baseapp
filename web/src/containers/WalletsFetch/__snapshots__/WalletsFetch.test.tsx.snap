// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WalletsFetch component should match snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Object {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "onStateChange": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <Connect(withRouter(Connect(WalletsFetchComponent))) />
</ContextProvider>
`;
